constants.go

```diff
@@ -252,6 +255,9 @@ const (
 	// ComponentWindowsDesktop is a Windows desktop access server.
 	ComponentWindowsDesktop = "windows_desktop"
 
+	// ComponentBot is a certificate renewal bot.
+	ComponentBot = "bot"
```

Comment:
What's the difference between `bot` and `tbot`? Do we need them both?

Reply:
D'oh, nope!

I definitely didn't just forget that we already had a constant defined...